Bottom: cdacf6301c83f5feaca1594c997cbc47f13c054a
Top:    cda9ae4bdc21f55dd0dd98e4a20fbbc556f814b3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-08 23:30:31 +0100

Refresh of hid-gtk-separate-out-board-fli

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index c106690..3b9fd5e 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1450,7 +1450,7 @@ currently within the window already.
 %end-doc */
 
 static int
-Center(int argc, char **argv, int x, int y)
+Center(int argc, char **argv, int pcb_x, int pcb_y)
 {
   GdkDisplay *display;
   GdkScreen *screen;
@@ -1465,7 +1465,7 @@ Center(int argc, char **argv, int x, int y)
   widget_x = gport->width / 2;
   widget_y = gport->height / 2;
 
-  ghid_pan_view_abs (x, y, widget_x, widget_y);
+  ghid_pan_view_abs (pcb_x, pcb_y, widget_x, widget_y);
 
   /* Now move the mouse pointer to the place where the board location
    * actually ended up.
@@ -1473,7 +1473,7 @@ Center(int argc, char **argv, int x, int y)
    * XXX: Should only do this if we confirm we are inside our window?
    */
 
-  ghid_pcb_to_event_coords (x, y, &widget_x, &widget_y);
+  ghid_pcb_to_event_coords (pcb_x, pcb_y, &widget_x, &widget_y);
   gdk_window_get_origin (gport->drawing_area->window, &offset_x, &offset_y);
 
   pointer_x = offset_x + widget_x;
